components/table/TableHead.js

```diff
@@ -5,7 +5,8 @@ const factory = (Checkbox) => {
     let selectCell;
     const contentCells = Object.keys(model).map((key) => {
       const name = model[key].title || key;
-      return <th key={key}>{name}</th>;
+      const width = model[key].width;
+      return <th key={key} style={{width}}>{name}</th>;
```

Comment:
It **is** using a style tag, the width is just the name of the variable in the data model.  Maybe widthStyle would be better.  It should definitely be applied to the body too, I can change that.